src/templates.yaml

```diff
@@ -39,4 +39,7 @@ tweet_noResult:  |
     Du hast kein Resultat oder es ist 0. Starte den Test erneut.
 
 measure: |
-    Ich starte eine Messung. Bitte frag mich in einer Minute nach dem Ergebnis.
\ No newline at end of file
+    Ich starte eine Messung. Bitte frag mich in einer Minute nach dem Ergebnis.
+
+stats_text:  |
+  Ich habe mir deine {{ type }} Resultate von heute angesehen. Mir liegen {{ count }} Resultate vor. Dein maximal Wert liegt bei {{ max }} und dein Minimum bei {{ min }}. Deine durchschnittliche Geschwindigkeit liegt heute bei {{ avg }}.
```

Comment:
@philipheimboeck I fixed it and added a new `stats_text` variation for the `ping` case.